src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './variables';
+
+.page {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
It's better to move font related styles to a separate file or at least to the top of this file. This will make it easier to manage your styles.